### ISSUE TITLE
did you find some new about repeated letter of result 

### ISSUE THREAD

**Issue by goodtogood** [novioleo/crnn.mxnet#2]
hi, there
did you find some new about repeated letter of result ?

**Comment by novioleo** [comment]
yes.these days i was working on optimize network,make it much more generic on scene text.
this weekend i will upload my new network,you can have a test,tks for your attention.

**Comment by goodtogood** [comment]
great to hear that , I'll test it when I'm free

**Comment by goodtogood** [comment]
actually I found some new problem about current code.
after 76 epochs train and validation accuracy can be about 0.96-0.99,
but the accuracy is just about  0.52 on **training set** ,
overfitting or some hidden bug?

**Comment by novioleo** [comment]
overfitting~~
